devices/webcam/setup.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+# Copyright 2004-present Facebook. All Rights Reserved.
+
+from setuptools import find_packages, setup
+
+setup(
+    name = "pose_vis",
+    version = "1.0.0",
+    description = "Pose visualization with LabGraph and MediaPipe",
+    packages = find_packages(),
+    python_requires = ">=3.8",
+    install_requires = [
+        # On Windows, LabGraph 2.0.1 gives a ModuleNotFound error relating to 'buck_ext'
```

Comment:
Alternatively, operating system can be checked (e.g. whether it's mac, win, or linux) to determine which install requires to be used (e.g. using os.name).